commit a7ed05478eb200f6c909e11b82e3809a1019aa5a
Author: Frank Schreiner <schreiner@suse.de>
Date:   Mon Feb 1 08:03:38 2021 +0100

    sort options in bash_completion
    
    without this patch the output of "<command> bash_completion" differs between
    each call. This breaks reproducible builds like in Debian or SUSE, because,
    even without code changes, the content of a built package (deb or rpm) which
    include a file containing the bash_completion differs on each build.
    See also https://wiki.debian.org/ReproducibleBuilds.
    
    This patch sorts the options before generating the output of bash_completion
    to generate reproducible results.

diff --git a/lib/MooseX/App/Plugin/BashCompletion/Command.pm b/lib/MooseX/App/Plugin/BashCompletion/Command.pm
index 247ee6f..311cc2f 100644
--- a/lib/MooseX/App/Plugin/BashCompletion/Command.pm
+++ b/lib/MooseX/App/Plugin/BashCompletion/Command.pm
@@ -62,7 +62,7 @@ EOT
     foreach my $command (sort keys %command_map) {
         $syntax .= "_${prefix}_macc_${command}() {\n    _${prefix}_compreply \"";
         #$syntax .= join(" ", @{$data->{parameters}});
-        $syntax .= join(" ", @{$command_map{$command}->{options}});
+        $syntax .= join(" ", sort @{$command_map{$command}->{options}});
         $syntax .= "\"\n}\n\n";
     }
 
